docs/pages/includes/install-linux.mdx

```diff
@@ -157,41 +198,74 @@ Use the appropriate commands for your environment to install your package.
   </Tabs>
   </TabItem>
   <TabItem label="Cloud" scope="cloud">
-
-  In the example commands below, update `$SYSTEM-ARCH` with the appropriate
-  value (`amd64`, `arm64`, or `arm`). All example commands using this variable
-  will update after one is filled out.
-
   <Tabs>
-  <TabItem label="Debian/Ubuntu (DEB)" scope="cloud">
+  <TabItem label="Debian/Ubuntu (apt)" scope="cloud">
 
-  After Downloading the `.deb` file for your system architecture, install it with
-  `dpkg`. The example below assumes the `root` user:
+  Add the Teleport repository to your repository list:
 
   ```code
-  $ dpkg -i ~/Downloads/teleport-ent_(=cloud.version=)_<Var name="$SYSTEM-ARCH"/>.deb
-  Selecting previously unselected package teleport-ent.
-  (Reading database ... 30810 files and directories currently installed.)
-  Preparing to unpack teleport-ent_(=cloud.version=)_$SYSTEM_ARCH.deb ...
-  Unpacking teleport-ent (=cloud.version=) ...
-  Setting up teleport-ent (=cloud.version=) ...
+  # Download Teleport's PGP public key
+  $ sudo curl https://apt.releases.teleport.dev/gpg \
+  -o /usr/share/keyrings/teleport-archive-keyring.asc
+  # Source variables about OS version
+  $ source /etc/os-release
+  # Add the Teleport APT repository for v(=teleport.major_version=). You'll need to update this
+  # file for each major release of Teleport.
```

Comment:
When using `stable/cloud` I think the idea is that you'll be upgraded across major versions automatically in line with whatever the current cloud release is, so this won't apply - right @r0mant @ptgott?

As such, we shouldn't reference `teleport.major_version` here or advise that people need to update the file every time there's a new release.

Reply:
fixed